src/httpclient.c

```diff
@@ -224,23 +243,24 @@ static void tunnel_on_udp_sock_read(h2o_socket_t *sock, const char *err)
 
     /* send the datagram directly or encapsulated on the stream */
     if (udp_write != NULL) {
-        h2o_iovec_t datagram = h2o_iovec_init(buf, rret);
+        h2o_iovec_t datagram = h2o_iovec_init(buf, rret + context_id_len);
```

Comment:
```suggestion
        h2o_iovec_t datagram = h2o_iovec_init(buf, context_id_len + rret);
```
Nothing more than style, but can we list the two in the order they appear?

Reply:
Done.